robotbase/src/main/java/com/team2357/frc2022/RobotContainer.java

```diff
@@ -35,15 +37,30 @@
   private final IntakeDriveControls m_driverControls;
   private final GunnerControls m_gunnerControls;
 
+  private final RobotArduino m_arduinoSensor;
+
+
```

Comment:
Typically we only have one blank line between sets of variables, or between functions. Not a big deal, just for your information.